selfdrive/boardd/boardd.cc

```diff
@@ -165,27 +176,116 @@ bool usb_connect() {
   }
 
   connected_once = true;
-  panda = tmp_panda.release();
+  if (tmp_panda->hw_type != cereal::PandaState::PandaType::UNO && tmp_panda->hw_type != cereal::PandaState::PandaType::DOS) {
+    panda_aux = tmp_panda.release();
+  } else {
+    panda = tmp_panda.release();
+  }
   return true;
 }
 
 // must be called before threads or with mutex
 static bool usb_retry_connect() {
-  LOGW("attempting to connect");
-  while (!do_exit && !usb_connect()) { util::sleep_for(100); }
+  
+  LOGW("attempting to find boards");
+  size_t detect_count = 0;
```

Comment:
No need for temp variable. Just use `pandas_detected.size()`

Reply:
I need it as I am removing from list of detected pandas, so line 203 will fail if I will use pandas_detected.size()